src/clj/rems/handler.clj

```diff
@@ -136,8 +115,17 @@
 
 (defn app-routes []
```

Comment:
The end result is better than before but this is perhaps slightly confusing route definition still. I don't have clear ideas on how to improve yet but the things are not exactly at the same level of abstraction (inlined stuff, wrappers, vars and not vars)

Reply:
Yeah but now the complexity is visible. I plan to simplify it further in the future (get rid of entitlements/routes for example)